chemdataextractor/model.py

```diff
@@ -460,6 +469,9 @@ def merge_contextual(self, other):
             for item in self[k]:
                 # print('item: %s' % item)
                 for other_item in other.get(k, []):
+		    #RBT Problem with doi: ma301230y. Had to add following check
+		    if (isinstance(other_item,unicode) == True):
+		        continue
```

Comment:
Seems this problem occurs when it is attempted to merge the 'contextual' information from a compound record into another one, but the 'contextual' record also happens to contain a name/label/role. This can happen when merging in information from table footnotes or caption - the idea is to merge e.g. a solvent into the property in the table cell that references it. But sometimes there is an actual compound named in the footnote. This fix seems to work well - just ignoring the string-based properties - which are name/label/role. However, `unicode` is python 2 only - this needs to be `six.text_type` for python 2 and 3 compatibility. In future, this whole `merge_contextual` method needs refactoring to be more agnostic of the model schema.